### ISSUE TITLE
Exporting an omr to mscz fails

### ISSUE THREAD

**Issue by B0GDAN** [Audiveris/audiveris#733]
Hello!

I've merged two omr files into one as my score has 2 pages but exporting to musescore fails with error:

Error in ExportBookTask java.lang.NullPointerException: Cannot invoke "java.util.List.iterator()" because the return value of "org.audiveris.omr.score.Score.getLogicalParts()" is null

Any ideas if there is a workaround available?

Thanks!

**Comment by hbitteur** [comment]
@B0GDAN 
Could you tell us more about the actions you have made.

In particular, have you merged the separate files and then transcribed the merged images (this is the normal case)?
Or have you transcribed each  image separately and then merged the transcriptions?

I suspect the second case, but please confirm.
A workaround might be to relaunch the transcription (from scratch) on the resulting merged file.

**Comment by hbitteur** [comment]
> If merging is needed before transcription this works with multi-page PDFs and TIFFs or single PNGs where the entire score is contained in one big image, correct?

You are totally right. The sources for merge can be of different formats (PDF, TIFF, PNG, JPG, ...) and kinds (multi vs single)

[Sorry for the late reply, I was hiking for the week.]
